### PR TITLE
Fix version_added of `isIntersecting`

### PR DESCRIPTION
**Pull request by MoyuScript**
#### Summary

Actually, `IntersectionObserverEntry.isIntersecting` was added after Chrome/58

See: <https://github.com/Fyrd/caniuse/blob/main/features-json/intersectionobserver.json#L222>

#### Test results and supporting details

I only tested this feature on Chrome/55

image

### CODE REVIEW

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thanks for this PR, @MoyuScript. I see it's your first to the project—welcome!

I confirmed this (via https://mdn-bcd-collector.appspot.com/tests/api/IntersectionObserverEntry/isIntersecting in Chrome 57 and 58). Happy to have this corrected.

It's likely that the other Chromium-derived browsers are impacted, however. I'd suggest the following changes:

Opera: 45
Opera for Android: 43
Samsung Internet: 7
WebView Android: 58

File: api/IntersectionObserverEntry.json [hunk 1]
```diff
@@ -280,10 +280,10 @@
               "version_added": "12.2"
             },
             "samsunginternet_android": {
-              "version_added": "5.0"
+              "version_added": "7"
```
Comment (queengooborg) [inline api/IntersectionObserverEntry.json:283]:
```suggestion
              "version_added": "7.0"
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Thanks for the quick updates!  Just got a small change to make, and this'll be all ready!

**Review by queengooborg — APPROVED** [review]
Thank you, looking good to me now!

**Pull request merged**